cli/tests/it/cmd.rs

```diff
@@ -1054,7 +1054,7 @@ forgetest_async!(check_broadcast_log, |prj: TestProject, cmd: TestCommand| async
     let run_log = re.replace_all(&run_log, "");
 
     // Ignore timestamp with `-11` since it changes inbetween runs.
-    assert!(&fixtures_log[..(fixtures_log.len() - 11)] == &run_log[..(run_log.len() - 11)]);
+    assert!(&fixtures_log[..(fixtures_log.len() - 12)] == &run_log[..(run_log.len() - 12)]);
```

Comment:
what's this magic number btw? :)

at least the doc needs to be updated as well

Reply:
It was trimming the end of the strings to ignore the timestamp because it is the last key of the JSON file. I actually did not need to change it, but I did because originally I misunderstood what it does.

Anyway, I know think it's slightly better with my last commit. I expanded the regex which was already used in this test to remove the blockhash to also remove the timestamp. This has the benefit of continuing to work in case:
1. the order to keys change in the JSON output
2. Foundry is still kicking in 2286 (because by then the unix timestamp in seconds would be more than 10 digits and we would need to do -12 on the string ;) )